torch_geometric/utils/rag/feature_store.py

```diff
@@ -137,7 +137,8 @@ def load_subgraph(
         # if it supported edge features
         edge_id = sample.edge
         edge_attr = self.edge_attr[edge_id]
-        return Data(edge_attr=edge_attr, edge_idx=edge_id)
+        edge_idx = torch.stack([sample.global_row, sample.global_col], dim=0)
```

Comment:
can you explain why we need this change

Reply:
if you need to replace edge_idx= torch.stack([sample.global_row, sample.global_col], dim=0)
then do we no longer need line 138?
edge_id = sample.edge
if its unused should drop it, but we definitely want to make sure accuracy remains safe after these changes